main.go

```diff
@@ -307,21 +364,21 @@ func runBaseTests(b base.Base, odometry movementsensor.MovementSensor, minLinVel
 		failedTests = append(failedTests, fmt.Sprintf("%v: %v", b.Name().ShortName(), storeErr))
 	}
 
-	// Spin: distance = 30 deg, speed = 15 deg/sec
-	if err := spinTest(b, odometry, 30, 15, false, f, f2); err != nil {
-		storeErr := fmt.Errorf("error spinning for 30 deg at 15 deg/sec, err = %v", err)
+	// Spin: distance = 40 deg, speed = 20 deg/sec
+	if err := spinTest(b, odometry, 40, 20, false, f, f2); err != nil {
+		storeErr := fmt.Errorf("error spinning for 40 deg at 20 deg/sec, err = %v", err)
```

Comment:
similarly wondering if its worth making the numbers in these strings variables, maybe by moving the fmt.Error into spinTest

Reply:
looking through this again and remembering why I didn't do it. i wanted to leave this logging outside the actual test so that if any other error occurs and the function returns early, it's still documented what test that happened in. for example if the actual `Spin` call in `spinTest` errors, the function will return before getting to the formatted error we expect, but I think it could still be helpful to know what spin test that was from if that makes sense.

Reply:
but I can still format this error so that 40 and 20 etc. aren't hardcoded into the string